### ISSUE TITLE
Add more to the README.md how to best setup your own environment. 

### ISSUE THREAD

**Issue by cpreston321** [zernonia/simple-log#6]
The `docs` I think need more docs on how to best setup your own instance and all the dependancies necessary and scripts

**Comment by zernonia** [comment]
I couldn't agree more. I will get to it soon!

**Comment by cpreston321** [comment]
@zernonia I was trying to add a feature yesterday, I found my-self trying to setup my own in order to get the app in a functional state.

**Comment by zernonia** [comment]
Yeah there's quite a number of steps to set it up for local testing. Thank you for so actively reporting the bug lol 😂
